.github/dependabot.yml

```diff
@@ -10,6 +10,8 @@ updates:
       - "area:dependabot"
     ignore:
       - dependency-name: "MessagePack" # Locked at a version that supports our net452 build target
+      - dependency-name: "*" # Ignore minors and patches for all integrations, majors change the signature
+        update-types: ["version-update:semver-minor", "version-update:semver-patch"]
```

Comment:
I agree on ignoring patches (gets rid of the obnoxious AWSSDK updates), but I think we should still keep track of minor versions since we do not always instrument public API's. Can we ignore just patches for now and later drop minor if it's still too much noise?

Reply:
Done